hack/jenkins/e2e.sh

```diff
@@ -850,9 +850,10 @@ fi
 if [[ "${USE_KUBEMARK:-}" == "true" ]]; then
   export RUN_FROM_DISTRO=true
   NUM_MINIONS_BKP=${NUM_MINIONS}
-  # We need to unset NUM_MINIONS to be able to use kubemark default settings.
-  unset NUM_MINIONS
+  MASTER_SIZE_BKP=${MASTER_SIZE}
   ./test/kubemark/stop-kubemark.sh
+  NUM_MINIONS=1000
```

Comment:
This will not work. If we want to have two suites:
kubemark-gce and kubemark-gce-scale
we just need to introduce another parameter and pass it to here.